app.js

```diff
@@ -1,11 +1,18 @@
-const express = require('express');
-let http = require('http');
+const express = require("express");
+let http = require("http");
 let app = express();
-app.use(express.static('dist'));
+app.use("/summary", express.static(__dirname + "/dist"));
```

Comment:
this should be root surely?

Reply:
If you don't map `/summary` to the `dist` folder then Express don't work properly. Try running `yarn setup && yarn start` on your branch and then go to the summary page. It doesn't work. We've got to manually map it which is rubbish but hopefully temporary.